pandas/core/dtypes/inference.py

```diff
@@ -263,7 +263,8 @@ def is_list_like(obj):
     """
 
     return (hasattr(obj, '__iter__') and
-            not isinstance(obj, string_and_binary_types))
```

Comment:
do this with an or between iter and collections....

Reply:
done